pkgs/by-name/li/limo/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+  pkg-config,
+
+  jsoncpp,
+  libarchive,
+  libcpr,
+  libloot,
+  pugixml,
+
+  libsForQt5,
+
+  withUnrar ? true,
+  unrar, # has an unfree license
```

Comment:
Thoughts on defaulting to true vs false?

Having unrar enabled by default means we have full functionality, but need to set allowUnfree in order to use the package :thinking: